dbms/src/Flash/Pipeline/Pipeline.cpp

```diff
@@ -26,6 +26,8 @@
 #include <Interpreters/Settings.h>
 #include <tipb/select.pb.h>
 
+#include <magic_enum.hpp>
+
```

Comment:
do we need this header?

Reply:
To use `magic_enum::enum_name` in https://github.com/pingcap/tiflash/pull/7474/files#diff-dce9669ca981e1c7ff8dd7f7f327469a65f85c31bc7fc9aa15a4dd6a9a46ea58R281-R283.